workspaces/diff-engine/src/events/mod.rs

```diff
@@ -102,3 +106,33 @@ impl From<avro_rs::Error> for EventLoadingError {
     EventLoadingError::Avro(err)
   }
 }
+
+// EventContext
+// ------------
+
+#[derive(Deserialize, Debug, PartialEq, Serialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct EventContext {
+  client_id: String,
+  client_session_id: String,
+  client_command_batch_id: String,
+  created_at: String,
+}
+
+impl From<CommandContext> for EventContext {
+  fn from(command_context: CommandContext) -> Self {
+    Self {
+      client_id: command_context.client_id,
+      client_session_id: command_context.client_session_id,
+      client_command_batch_id: command_context.client_command_batch_id,
+      created_at: command_context
+        .created_at
+        .with_timezone(&Local)
+        .to_rfc3339(),
```

Comment:
learned something new today
```
fn parse_from_rfc3339(s: &str) -> ParseResult<DateTime<FixedOffset>>[−]
Parses an RFC 3339 and ISO 8601 date and time string such as 1996-12-19T16:39:57-08:00, then returns a new DateTime with a parsed FixedOffset.

Why isn't this named parse_from_iso8601? That's because ISO 8601 allows some freedom over the syntax and RFC 3339 exercises that freedom to rigidly define a fixed format.```

Reply:
I learned the exact same thing about a week ago, after looking to `to_iso8601` 😅